performance-tools/performance-cluster/chart/kyma-cluster/templates/pod.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: setup-{{ template "kyma-cluster.fullname" . }}
+  labels:
+    app: {{ template "kyma-cluster.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    grade: "{{ .Values.labels.grade}}"
+    "kyma-project.io/kyma-cluster": "{{ .Values.labels.grade}}"
+spec:
+  serviceAccountName: setup-{{ template "kyma-cluster.fullname" . }}
+  restartPolicy: Never
+  containers:
+  - name: kyma-cluster
+    image: "{{ .Values.containerRegistry.path }}/{{ .Values.image.dir }}performance-cluster:{{ .Values.image.version }}"
+    imagePullPolicy: "{{ .Values.image.pullPolicy}}"
+    command: ["/scripts/kyma-gke-cluster.sh"]
+    env:
+    - name: APP_MAX_CONCURRENCY_LEVEL
+      value: "1"
+    - name: APP_LOGGER_LEVEL
+      value: "info"
+    - name: INPUT_CLUSTER_NAME
+      value: "{{ .Values.global.clusterName }}"
+    - name: DOCKER_PUSH_REPOSITORY
+      value: "{{ .Values.global.dockerPushRepository }}"
+    - name: DOCKER_PUSH_DIRECTORY
+      value: "{{ .Values.global.dockerPushDirectory }}"
+    - name: KYMA_PROJECT_DIR
+      value: "{{ .Values.global.kymaProjectDir }}"
+    - name: CLOUDSDK_CORE_PROJECT
+      value: "{{ .Values.global.cloudCoreProject }}"
+    - name: CLOUDSDK_COMPUTE_REGION
+      value: "{{ .Values.global.cloudComputeRegion }}"
+    - name: CLOUDSDK_COMPUTE_ZONE
+      value: "{{ .Values.global.cloudComputeZone }}"
+    - name: CLOUDSDK_DNS_ZONE_NAME
+      value: "{{ .Values.global.cloudDndZoneName }}"
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: "{{ .Values.global.cloudAppCredentials }}"
+    - name: KYMA_ARTIFACTS_BUCKET
+      value: "{{ .Values.global.kymaArtifactsBucket }}"
```

Comment:
K8s configmap will be a good idea for the following:
- CLOUDSDK_CORE_PROJECT
- CLOUDSDK_COMPUTE_REGION
- KYMA_ARTIFACTS_BUCKET
- DOCKER_PUSH_DIRECTORY
-